### PR TITLE
Fixes to media-libs/allegro (reopen)

### PR DESCRIPTION
**Pull request by winterheart**
Version bump to 5.2.4.0 (#623798) Security fixes (#540470) Added multilib support for 4.2, EAPI 6

### CODE REVIEW

**Comment by a17r** [comment]
In your `4.4.2` revbump (check and fix git message!), please sync with the recent changes to -r1 ebuild. Please also diff `5.2.4` over current `5.0.11` not to miss any changes.

File: media-libs/allegro/allegro-5.2.4.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	)
+
+	cmake-multilib_src_configure
+}
+
+src_install() {
+	cmake-multilib_src_install
+
+	dodoc CHANGES-5.0.txt
+	dodoc -r docs/html/refman/*
```
Comment (Zlogene) [inline media-libs/allegro/allegro-5.2.4.0.ebuild]: unneeded glob, `-r` handles recursively

**Comment by winterheart** [comment]
Done.

**Comment by a17r** [comment]
> Done.

hmm not quite, I still see many changes back from current to out-of-tree state ebuild versions in both 4 and 5.

**Comment by a17r** [comment]
I've pushed part of this PR: Your format-security fix and the 5.2.4.0 version bump (there were too many other changes to do there so I did not bother you with more review).

Now, before we move on to 4.4.2 multilib support, is there anything in tree that would profit from the added complexity in the ebuild?

**Comment by winterheart** [comment]
My intention for multilib is bump media-libs/dumb to actual version 2.0 and remove old media-libs/aldumb (new version already has integrated support for allegro 4.2).

**Comment by a17r** [comment]
Then please update your PR with `git pull --rebase gentoo master` (assuming `gentoo` is your upstream remote), only the 4.4.2 revbump should remain after resolving the conflicts.

Take a good look at cc402fa4548fc878e06c5fa778c519feeee27334 to make sure you caught all the changes for your ebuild.

**Comment by a17r** [comment]
ping, do you still plan to work on this?

**Comment by winterheart** [comment]
Too much changes. Reworked on new PR #11244

**Pull request closed without merging**